Code/Framework/AzQtComponents/AzQtComponents/Actions/ActionManagerConstants.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+static constexpr const char s_actionManagerToggleKey[] = "/O3DE/ActionManager/EnableNewActionManager";
```

Comment:
Would it be less invasive to use a util function to check the registry setting instead of adding m_enableNewActionManager in multiple places?

Reply:
+1 - could do something similar to what we have in EditorViewportSettings (and ViewportSettings.h/cpp) to query the setreg value without adding new member variable state?

Reply:
I was thinking about it but I was going towards the direction of having a singleton handler. What you mean is more of a static helper?

I guess that works better, yes. Was thinking there may be benefits in caching the value, but since it's only checked at startup that's likely a non-issue. I will add this change and send an update :D

Reply:
Awesome stuff thanks Danilo, I was just thinking something dead simple like...

```
static constexpr const char ActionManagerToggleKey[] = "/O3DE/ActionManager/EnableNewActionManager";

bool NewActionManagerEnabled()
{
    return GetRegistry(EnableNewActionManager, false);
}
```

Where `GetRegistry` is a helper in `AzToolsFramework\Viewport\ViewportSettings.h`